### ISSUE TITLE
FF: script message generated when double click from wordlist

### ISSUE THREAD

**Issue by monzug** [alpheios-project/alpheios-core#136]
build 3.0.0.91: I got the yellow browser message (a script is running) in FF. to repro double-click on paenitere   or   sum    in  https://it.wikipedia.org/wiki/Verbi_latini_deponenti_e_semideponenti      
then go to inflection table then go to wordlist and limit by this session. then double click on the same word. the yellow message will be generated.

**Comment by irina060981** [comment]
@balmas and @monzug , may be it is worth to retest it after all made refactoring?

**Comment by balmas** [comment]
Yes I agree.  Assigning to @monzug for some testing on the latest dev build.

**Comment by monzug** [comment]
I got the yellow message when double clicking on paenitere from wordlist.
FF:  Alpheios Reading Tools 3.3.3 build 20210122645
Alpheios Components 3.3.3-20210122585
